### PR TITLE
Insert Nft previews

### PR DESCRIPTION
**Pull request by kiroj305**
- insert nft preview between `upload` & `edit`
- code refactor

### CODE REVIEW

**Comment by kjawadDeveloper2** [comment]
Have to check this on tablet and then will merge it .

File: lib/screens/home_screen.dart [hunk 1]
```diff
@@ -142,31 +151,29 @@ class _HomeScreenState extends State<HomeScreen> {
...
-                tabletScreen: (BuildContext context) => const VerticalSpace(30),
+                tabletScreen: (context) => const VerticalSpace(30),
               ),
               Expanded(
                 child: PageView(
                   controller: _pageController,
                   physics: const NeverScrollableScrollPhysics(),
                   onPageChanged: (int page) {
                     _currentPage.value = page;
+                    if (page <= 2) {
```
Comment (mustafapylons) [inline lib/screens/home_screen.dart]: constants "2" and "3" can be in variable.
switch/case might be more efficient here.

File: lib/screens/preview_screen.dart [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+}
+
+class _PreviewScreenState extends State<PreviewScreen> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Consumer<EaselProvider>(
+        builder: (_, provider, __) => Stack(
+          children: [
+            if (provider.nftFormat.format == kImageText) ...[ImageWidget(file: provider.file!)],
```
Comment (mustafapylons) [inline lib/screens/preview_screen.dart:28]: is there a better way to write these if statements?

File: lib/utils/constants.dart [hunk 3]
```diff
@@ -78,14 +78,14 @@ const kSizeText = "Size";
...
-const kPreviewNFTText = "Preview NFT";
+const kPreviewNFTText = "Preview your NFT";
 const kPreview3dModelText = "Click here to preview\nyour selected 3D Model";
 const kMintMoreText = "Mint More";
 const kGoToWalletText = "Go to Wallet";
 const kChooseNFTFormatText = "Choose your NFT format";
 const kUploadNFTText = "Upload your NFT file";
 const kEditNFTText = "Edit your NFT Details";
-const kStartText = "Start";
+// const kStartText = "Start";
```
Comment (mustafapylons) [inline lib/utils/constants.dart]: is this commented code necessary

**Review by mustafapylons — COMMENTED** [review]
Few comments, otherwise code is good.

**Comment by kiroj305** [comment]
@kjawadDeveloper2 , this has been tested on tablet by Tanya today. Please refer to this video https://we.tl/t-lKXClzzoPK

**Comment by kiroj305** [comment]
@mustafapylons , issues pointed out during your review were fixed

**Review by mustafapylons — APPROVED** [review]
Looks good to me now.

**Review by kjawadDeveloper2 — APPROVED** [review]
Looks good to me .

**Pull request merged**